src/libunwind/build.rs

```diff
@@ -4,11 +4,11 @@ fn main() {
     println!("cargo:rerun-if-changed=build.rs");
     let target = env::var("TARGET").expect("TARGET was not set");
 
-    if cfg!(feature = "llvm-libunwind") &&
+    if cfg!(all(not(bootstrap), feature = "llvm-libunwind")) &&
```

Comment:
Can you add a comment that explains the `not(bootstrap)` part is due to #62286, and can be removed once that is in beta?